Microsoft.Toolkit.Mvvm/Microsoft.Toolkit.Mvvm.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <Title>Windows Community Toolkit Mvvm .NET Standard</Title>
+    <Description>
+      This package includes Mvvm .NET Standard code only helpers such as:
+      - Ioc: Provides a fast and minimalistic implementation of an IoC container.
```

Comment:
Don't forget to update the description here.

Reply:
Right! Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3229/commits/56dfbaae20927c2f0bce68728a73d651ef522d39.